src/DataWrangling/Bathymetry.jl

```diff
@@ -1,62 +1,33 @@
 module Bathymetry
```

Comment:
the name should be `module ETOPO` right?

Reply:
There are some bathymetry-related methods that don't necessarily belong in the ETOPO module, e.g., `regrid_bathymetry!`. These could be part of the DataWrangling module, e.g., gathered together in a file called `bathymetry_utils.jl` or something; they don't need their own module. And then we have a separate module for every dataset, e.g., ETOPO, GEBCO, etc.

Reply:
I think we want to have `Bathymetry` module for things related to manipulating bathymetry and then `ETOPO`, etc, for the data downloading

Reply:
That also works